wedding_covid_numbers.Rmd

```diff
@@ -48,7 +48,7 @@ us_deaths
 
 Looking at cases alone is not enough: case reporting is highly variable depending on testing protocols and resources. 
 
-Here, we look at total deaths since the beginning of the pandemic. The story is the same for New York, New Jersey, and Massachusetts. Pennsylvania, Illinois, Indiana, and Maryland make a motley crew walking the middle path, while the rest seem to be following Washington's trajectory. 
+Here, we look at total deaths since the beginning of the pandemic. The story is the same for New York, New Jersey, and Massachusetts - with Michigan actively _distancing_ (ha) from them. Pennsylvania, Illinois, Indiana, and Maryland make a motley crew walking the middle path, while the rest seem to be following Washington's trajectory. This "rest" sees roughly 10 times fewer death per inhabitants than the "top" states.
```

Comment:
```suggestion
Here, we look at total deaths since the beginning of the pandemic. The story is the same for New York, New Jersey, and Massachusetts - with Michigan actively _distancing_ (ha) from them. Pennsylvania, Illinois, Indiana, and Maryland make a motley crew walking the middle path, while the rest seem to be following Washington's trajectory. This "rest" is seeing roughly 10 times fewer deaths per inhabitants than the "top" states.
```